plugins/ecue.js

```diff
@@ -4,14 +4,14 @@ const colorNames = require('color-names');
 const xml2js = require('xml2js');
 const xmlbuilder = require('xmlbuilder');
 
+const defaults = require(path.join(__dirname, '..', 'fixtures', 'defaults'));
+
 module.exports.name = 'e:cue';
 module.exports.version = '0.2.0';
```

Comment:
ecue's output will not change, qlcplus's possibly does, as capabilities may be different.